### ISSUE TITLE
No data in NCBI for genes BRCA1, KRAS, EGFR

### ISSUE THREAD

**Issue by NadiraGaliamova** [epam/NGB#360]
Steps to reproduce:

1. Open NGB.
4. In the 'Positions' in the upper side type the EGFR gene.
5. Select EGFR gene.
6. Make sure 'GRCh37.NewRefSeqGenes' part of graphical data is present in the NGB browser.
7. Click on the transcript presented in 'GRCh37.NewRefSeqGenes' section.
8. Select 'Show Info'.
9. Switch to the NCBI tab.

Expected result:
The gene NCBI data is displayed.

Actual result:
"No Data Available. No result was were returned by external db" message is displayed.

Note: The same issue is reproduced for BRCA1, KRAS genes.

![NCBI](https://user-images.githubusercontent.com/64957299/105324745-da664400-5bdc-11eb-9194-1eaf3cee5e44.png)

**Comment by mike-miroliubov** [comment]
The issue looks to be caused by the limitations of NCBI API: these genes seems to be mentioned in lots of publications and NCBI API has a limitation of 500 IDs maximum per request. 

An attempt to access https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?retmode=json&query_key=1&WebEnv={webEnv} returns the following result: 
```
{
  "header": {
    "type": "esummary","version":"0.3"
  },
  "error": "Too many UIDs in request. Maximum number of UIDs is 500 for JSON format output."
}
```

I would say the issue is quite a general one, it is represented no only with that custom `GRCh37.NewRefSeqGenes.gtf.gz` file, but with a reference genes Homo_Sapiens.GRCh37.gtf.gz and for [GRCh38](http://ngb.opensource.epam.com/catgenome/#/GRCh38/17/43044295/43170245?tracks=%5B%7B%22b%22:%22GRCh38%22,%22p%22:%22RNASeq-chr22-SpliceJunctions%22,%22f%22:%22REFERENCE%22,%22projectIdNumber%22:5,%22h%22:20,%22s%22:%7B%22rt%22:false,%22rsfs%22:true,%22rsrs%22:false%7D%7D,%7B%22b%22:%22brain_th.bam%22,%22p%22:%22RNASeq-chr22-SpliceJunctions%22,%22f%22:%22BAM%22,%22projectIdNumber%22:5,%22h%22:250,%22s%22:%7B%22a%22:true,%22aa%22:true,%22c%22:%22noColor%22,%22c1%22:true,%22d%22:true,%22g1%22:%22default%22,%22i%22:true,%22m%22:true,%22r%22:%221%22,%22s1%22:false,%22s2%22:true,%22s3%22:false,%22v1%22:false,%22cdm%22:%22Bar%20Graph%22,%22cls%22:false,%22csm%22:%22default%22%7D%7D,%7B%22b%22:%22PIK3CA-E545K.bam%22,%22p%22:%22PIK3CA-E545K-Sample%22,%22f%22:%22BAM%22,%22projectIdNumber%22:4,%22h%22:250,%22s%22:%7B%22a%22:true,%22aa%22:true,%22c%22:%22noColor%22,%22c1%22:true,%22d%22:true,%22g1%22:%22default%22,%22i%22:true,%22m%22:true,%22r%22:%221%22,%22s1%22:false,%22s2%22:true,%22s3%22:false,%22v1%22:false,%22cdm%22:%22Bar%20Graph%22,%22cls%22:false,%22csm%22:%22default%22%7D%7D,%7B%22b%22:%22PIK3CA-E545K.cram%22,%22p%22:%22PIK3CA-E545K-Sample%22,%22f%22:%22BAM%22,%22projectIdNumber%22:4,%22h%22:250,%22s%22:%7B%22a%22:true,%22aa%22:true,%22c%22:%22noColor%22,%22c1%22:true,%22d%22:true,%22g1%22:%22default%22,%22i%22:true,%22m%22:true,%22r%22:%221%22,%22s1%22:false,%22s2%22:true,%22s3%22:false,%22v1%22:false,%22cdm%22:%22Bar%20Graph%22,%22cls%22:false,%22csm%22:%22default%22%7D%7D,%7B%22b%22:%22GRCh38_Genes%22,%22f%22:%22GENE%22,%22l%22:true,%22p%22:%22%22,%22h%22:100,%22s%22:%7B%22g%22:%22collapsed%22%7D%7D%5D) too.

We can work around this issue by including a *retmax=500* parameter into the URL: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?retmode=json&query_key=1&WebEnv={webEnv}&retmax=500.

We can implement pagination with multiple requests (using incrementing *retstart* parameter). However, a number of publications more that 500 seems too much and not quire relevant for the user to absorb via NGB UI... So in this case we might just limit overall number of publications fetched to 500 and call it a day. @NadiraGaliamova and her team seems to be satisfied with this approach.
What do you think, Arleksandr @sidoruka?

**Comment by mike-miroliubov** [comment]
Fixed in version 2.6.0.2147
